raphtory/src/db/graph/views/filter/model/property_filter.rs

```diff
@@ -502,8 +578,17 @@ impl<M> PropertyFilter<M> {
         match meta.get_prop_id_and_type(name, is_static) {
             None => Ok(None),
```

Comment:
We should make this case an error and return Result<usize, GraphError> (see #2285)